docs/install_guides/pypi-openvino-dev.md

```diff
@@ -26,12 +26,20 @@ Before you start the installation, check the supported operating systems and req
 
 Use a virtual environment to avoid dependency conflicts. 
 
-To create a virtual environment, use the following command:
+To create a virtual environment, use the following commands:
+
+On Windows:
 ```sh
 python -m pip install --user virtualenv 
 python -m venv openvino_env
 ```
 
+On Linux and macOS:
+```sh
+python3 -m pip install --user virtualenv 
+python3 -m venv openvino_env
+```
+
 > **NOTE**: On Linux and macOS, you may need to type `python3` instead of
```

Comment:
If the instructions say to type python3 on Linux/macOS, the first part of this note seems unnecessary

Reply:
Yes, the feedback from the UX research says that "type python3 on Linux/macOS" is not obvious enough, so we need to explicitly provide the command. I kept the first sentence in the note just in case, now I think it's safe to remove.